prefect_gcp/workers/vertex.py

```diff
@@ -111,15 +113,15 @@ class VertexAIWorkerVariables(BaseVariables):
             "The type of accelerator to attach to the machine. "
             "See https://cloud.google.com/vertex-ai/docs/reference/rest/v1/MachineSpec"
         ),
-        default="NVIDIA_TESLA_K80",
+        example="NVIDIA_TESLA_K80",
     )
-    accelerator_count: int = Field(
+    accelerator_count: Optional[int] = Field(
         title="Accelerator Count",
         description=(
             "The number of accelerators to attach to the machine. "
             "See https://cloud.google.com/vertex-ai/docs/reference/rest/v1/MachineSpec"
         ),
-        default=0,
+        example=1,
```

Comment:
this is unrelated to the kill infra path, but something i found while testing this further

both the `accelerator_*` params are optional, and they have an XOR relationship - both have to exist, or both have to be null.  so, i swapped these to be optional + removed the keys from the job body validation

Reply:
im cool to add the default `None`, but with the current configuration, the UI looks like this:

<img width="809" alt="image" src="https://github.com/PrefectHQ/prefect-gcp/assets/19556538/d3ae8796-8e34-4440-8ace-ec50c7100f02">

Reply:
that's so weird, i would expect that as well.  but i just created a new work pool and started the worker, and:

```
$ prefect work-pool create 'a-new-pool' --type vertex-ai
Created work pool 'a-new-pool'.

$ prefect worker start -p a-new-pool
Discovered type 'vertex-ai' for work pool 'a-new-pool'.
Worker 'VertexAIWorker b1b27bbe-1070-4f5a-9ccc-f63a6e7adad1' started!
```

here are the work pool variables in the UI
![image](https://github.com/PrefectHQ/prefect-gcp/assets/19556538/74535541-1bd7-4300-bbde-42175c72d726)

Reply:
You wouldn't see the error until a flow run is prepared to be run because that's when a configuration object is created.

Reply:
yeah i didn't see it at all during my e2e testing, where i created a bunch of vertex jobs.  does it make a difference that these properties are on the variables class, not the configuration?

Reply:
btw i added the default `None` values in, this thread is more for my understanding / sanity

Reply:
🤦‍♂️ I didn't realize this is the variables class. No objects are ever created from this class, so you wouldn't see that error. It all makes sense now.

Reply:
ok whew, sanity restored